### PR TITLE
[RelatedArtists] iPad fixes

### LINKED ISSUE #152: [iPad] Related artists grid issues.
- [x] Fix trailing artist alignment
- [x] Decide: 3 or 4 columns?
- [ ] Fix: Different column size in orientation changes

## When there’s a single trailing artist it doesn’t align to the left:

This can be seen with the artist `alex-katz`.

simulator screen shot 02 jun 2016 14 41 50

## Should have 3 columns in each orientation

According to [the design spec](https://github.com/artsy/eigen/issues/1136), it should also have 3 columns in landscape orientation. I’m not sure what @sarahscott based the 4 columns on, maybe there’s another comment elsewhere?

## When loading ‘ABOUT’ tab in landscape orientation and then changing to portrait orientation changes related artists to 2 columns.

How this should be fixed kinda depends on whether or not there are always supposed to be 3 columns. _(See above.)_

simulator screen shot 02 jun 2016 14 50 40

### PR DESCRIPTION
**Pull request by mennenia**
This fixes #152, #154, #155. 

Current status: @sarahscott is awesome!

### CODE REVIEW

**Comment by mennenia** [comment]
I think the issue is the new imagewidth? 

I now got it to have 4 related artists on load when in landscape, but it still goes to only two columns on portrait. 

Logging shows that the related artists widths get recalculated from portrait to landscape (2 logs, as expected), but not when you start the app in landscape and then go to portrait (only 1 log, the initial one)

Unsure what's going on yet. Going to have dinner and will be back.

**Comment by sarahscott** [comment]
I think I got it!

ipadrotationsuccess

File: lib/components/artist/related_artists/index.js [hunk 1]
```diff
@@ -18,20 +18,34 @@ class RelatedArtists extends React.Component {
...
   layoutState() {
-    const window = Dimensions.get('window');
-    const isPad = window.width > 700;
-    const isPadHorizontal = window.width > 1000;
+    const width = Dimensions.get('window').width;
+    const isPad = width > 700;
+    const isPadHorizontal = width > 1000;
 
-    const columnCount = (isPad ? (isPadHorizontal ? 4 : 3) : 2);
+    // TODO: Document what these margins are based on.
```
Comment (alloy) [inline lib/components/artist/related_artists/index.js]: @sarahscott @mennenia Would love to have this sorted. They are really magic numbers right now. I’d also prefer to distinguish between side-margins and in-between margins and then calculate the column width based on that.

**Comment by alloy** [comment]
Alright, let the festivities begin! 🎉

File: lib/components/artist/related_artists/index.js (hunk 1 above)
Comment (mennenia) [inline lib/components/artist/related_artists/index.js]: Sounds good. I have zero clue about the margins T_T.

**Pull request merged**